code/modules/multiz/movement.dm

```diff
@@ -220,10 +220,20 @@
 
 
 /mob/living/carbon/human/can_fall(turf/below, turf/simulated/open/dest = src.loc)
-	// Special condition for jetpack mounted folk!
-	if (!restrained())
-		if (CanAvoidGravity())
```

Comment:
You would need to put back this `CanAvoidGravity()` check somewhere in here (if it's even used).